### ISSUE TITLE
Should `get_string` of `StringTableSection` return string instead of bytes?

### ISSUE THREAD

**Issue by JaySon-Huang** [eliben/pyelftools#73]
I tried to use 

    efile.get_section_by_name('.init')

to get the '.init' section but it return None. I found it that name is bytes instead of string in Python3.

so now it should use

    efile.get_section_by_name(b'.init')

to successfully get the right section. But I think that it is more easy and more readable way to get it by string.

What's more, the method name is `get_string` but not `get_cstring`

**Comment by eliben** [comment]
Yes, this is a good point. The strings should probably be decoded to strings.

AFAIU, the default DWARF encoding is ascii, unless a special setting exists that sets it to UTF-8.
